pyproject.toml

```
@@ -43,7 +43,7 @@ numpy = [
     {version = ">=1.22", python = ">=3.10"}
 ]
 scikit-learn = ">=0.22"
-lightgbm = ">=2.3, <=3.2.1"
+lightgbm = ">=2.3"
```

please leave it as it is:
```suggestion
lightgbm = ">=2.3, <=3.2.1"
```

---

Done